components/tracer/tests/integration/testdata/legacy_dual_runner/000001_initial_schema.up.sql

```
@@ -0,0 +1,317 @@
+    expression TEXT NOT NULL,
+    action decision_enum NOT NULL,
+    scopes JSONB NOT NULL DEFAULT '[]',
+    status rule_status_enum NOT NULL DEFAULT 'DRAFT',
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    activated_at TIMESTAMP WITH TIME ZONE,
+    deactivated_at TIMESTAMP WITH TIME ZONE,
+    deleted_at TIMESTAMP WITH TIME ZONE
+);
+
+CREATE INDEX IF NOT EXISTS idx_rules_status ON rules(status) WHERE deleted_at IS NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)